.github/workflows/funcbench.yml

```diff
@@ -0,0 +1,56 @@
+on: repository_dispatch
+name: Funcbench Workflow
+jobs:
+  run_funcbench:
+    name: Running funcbench
+    if: github.event.action == 'funcbench_start'
+    runs-on: ubuntu-latest
+    env:
+      AUTH_FILE: ${{ secrets.PROMBENCH_GKE_AUTH }}
+      BRANCH: ${{ github.event.client_payload.BRANCH }}
+      BENCH_FUNC_REGEX: ${{ github.event.client_payload.BENCH_FUNC_REGEX }}
+      CLUSTER_NAME: prombench
+      GITHUB_TOKEN: ${{ secrets.PERSONAL_TOKEN }}
+      GITHUB_ORG: prometheus
+      GITHUB_REPO: prometheus
+      LAST_COMMIT_SHA: ${{ github.event.client_payload.LAST_COMMIT_SHA }}
```

Comment:
can't this be fetched using `github.event.client_payload.PR_NUMBER` ? I mean once you know the PR number make a query to find out the latest commit she of this PR. This will remove the need for this variable.

Reply:
`LAST_COMMIT_SHA` is used for setting the commit status to Pending, to prevent it for being merged, like DCO check.
If we don't need this check, we can get rid of it.

Reply:
I think this is not to prevent a merge, but more for showing at the bottom of the page that there is a funcbnech running and if I am correct we need this.

Reply:
`LAST_COMMIT_SHA` is fetched from the pr number only but here it is done by comment-monitor  when it sees a comment as it is almost always required if we want to enable Github status towards the bottom of the PR